src/components/HomePage/PlayRadioButton.tsx

```diff
@@ -55,6 +60,16 @@ const PlayRadioButton = () => {
   const { radioActor } = audioService.getSnapshot().context;
   return (
     <div className={styles.container}>
+      <Button
+        href={getCoursesNavigationUrl()}
+        variant={ButtonVariant.Outlined}
+        className={styles.ctaButton}
+        size={ButtonSize.Small}
+        type={ButtonType.Secondary}
+        onClick={onCoursesClicked}
+      >
+        {t('common:courses')}
+      </Button>
```

Comment:
I don't think this is the best place for the courses button. I still don't know what would the best place be. But what I have in mind is to add it in this section:

![CleanShot 2024-01-16 at 19 41 52@2x](https://github.com/quran/quran.com-frontend-next/assets/58295120/35d718a7-df3f-4540-9c05-157107f3c201)

But maybe with a different color

Reply:
This was agreed on with Osama until we have a UX resource and we would re-design our nav bar to include it.